src/app/zap-templates/zcl/data-model/chip/matter-devices.xml

```diff
@@ -1957,72 +1957,111 @@ limitations under the License.
             <include cluster="Audio Output" client="false" server="true" clientLocked="false" serverLocked="false"></include>
             <include cluster="Low Power" client="false" server="true" clientLocked="false" serverLocked="false"></include>
             <include cluster="Wake on LAN" client="false" server="true" clientLocked="false" serverLocked="false"></include>
             <include cluster="Target Navigator" client="false" server="true" clientLocked="false" serverLocked="false"></include>
             <include cluster="Account Login" client="false" server="true" clientLocked="false" serverLocked="false"></include>
-            <include cluster="Content Launcher" client="false" server="true" clientLocked="false" serverLocked="false"></include>
+            <include cluster="Content Launcher" client="false" server="true" clientLocked="false" serverLocked="true"></include>
         </clusters>
     </deviceType>
-
     <deviceType>
         <name>MA-basic-videoplayer</name>
         <domain>CHIP</domain>
         <typeName>Matter Basic Video Player</typeName>
         <profileId editable="false">0x0103</profileId>
         <deviceId editable="false">0x0028</deviceId>
         <clusters lockOthers="true">
-            <include cluster="Identify" client="false" server="true" clientLocked="true" serverLocked="true">
-                <requireAttribute>IDENTIFY_TIME</requireAttribute>
-                <requireAttribute>IDENTIFY_TYPE</requireAttribute>
-                <requireCommand>Identify</requireCommand>
-                <requireCommand>IdentifyQuery</requireCommand>
-                <requireCommand>TriggerEffect</requireCommand>
-            </include>
             <include cluster="Descriptor" client="false" server="true" clientLocked="true" serverLocked="true">
                 <requireAttribute>DEVICE_TYPE_LIST</requireAttribute>
                 <requireAttribute>SERVER_LIST</requireAttribute>
                 <requireAttribute>CLIENT_LIST</requireAttribute>
                 <requireAttribute>PARTS_LIST</requireAttribute>
             </include>
-            <include cluster="Binding" client="true" server="false" clientLocked="false" serverLocked="true">
-                <requireAttribute>BINDING</requireAttribute>
-            </include>
             <include cluster="Media Playback" client="false" server="true" clientLocked="false" serverLocked="true"></include>
             <include cluster="Keypad Input" client="false" server="true" clientLocked="false" serverLocked="true"></include>
-            <include cluster="Application Launcher" client="false" server="true" clientLocked="false" serverLocked="false"></include>
             <include cluster="Media Input" client="false" server="true" clientLocked="false" serverLocked="false"></include>
             <include cluster="On/Off" client="false" server="true" clientLocked="false" serverLocked="false"></include>
```

Comment:
On/Off is mandatory for this device type.  This part was correct in previous versions of this PR.

I'm sorry, but I'm pretty tired of having to double-check every single line against the spec every single time because when things are fixed other things just get broken in the process.   Could you please go through and do that?  Carefully.  For every single line involved.  Then ask for review.

Alternately, do sepatate PRs for the separate device types; maybe that would allow things to converge more quickly in that fixes needed to some of them won't undo fixes for the others.

Reply:
Updated and I've double checked with:
- [CastingVideoClient.adoc](https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/device_types/CastingVideoClient.adoc)
- [CastingVideoPlayer.adoc](https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/device_types/CastingVideoPlayer.adoc)
- [BasicVideoPlayer.adoc](https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/device_types/BasicVideoPlayer.adoc)
- [ContentApp.adoc](https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/device_types/ContentApp.adoc)
- [Speaker.adoc](https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/device_types/Speaker.adoc)